### PR TITLE
fix(samples): fix column name in DagModel of Airflow workflow to clean up database

### LINKED ISSUE #6531: Airflow DB clean up DAG not working with recent versions
## In which file did you encounter the issue?
https://github.com/GoogleCloudPlatform/python-docs-samples/blob/9c71acbdf534defb400212ea68cf8359e3a3c4c3/composer/workflows/airflow_db_cleanup.py

### Did you change the file? If so, how?

-    "age_check_column": DagModel.last_scheduler_run,
+    "age_check_column": DagModel.last_parsed_time,

## Describe the issue

Starting from Airflow 2.0.2 `last_scheduler_run` column renamed into `last_parsed_time` in `DagModel`.
https://github.com/apache/airflow/blob/96656527fed515d8f39bbbec5d6a155faae2aaa2/CHANGELOG.txt#L369

### PR DESCRIPTION
**Pull request by kosteev**
## Description

Fixes #6531 

## Checklist
- [X ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [X ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Comment by leahecole** [comment]
@kosteev to double check do we need to update this for [Airflow 1](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/composer/airflow_1_samples/airflow_db_cleanup.py) as well?

**Review by leahecole — DISMISSED** [review]

**Comment by leahecole** [comment]
Hey @kosteev - I now see tests aren't passing. Are they passing for you locally? (See the [Testing Guidelines](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#testing-guidelines) for instructions on running tests locally)

**Comment by kosteev** [comment]
@leahecole I've bumped version of apache-airflow in requirements.txt file.

I wasn't able to run locally tests with nox, but I have installed pypi requirements* manually and ran py.test manually, and airflow_db_cleanup_test has passed successfully.

Can you, please trigger Kokoro. I believe it has to be triggered manually, as I do not see it is executed automatically.

**Comment by kosteev** [comment]
I see it failed again. Trying actually to run all tests locally and fix.

**Comment by kosteev** [comment]
Ok.
I was able to setup local environment and run tests with nox.

I have updated constraints.txt file to have constraints (I didn't realized we have it) to Airflow 2.1.2 constraints (as I bumped it in requirements.txt) and it was the reason for failing tests with new Airflow version. Now hadoop_tutorial_test.test_dag_import passes for me locally.

Please, run Kokoro again.

**Comment by kosteev** [comment]
Side question: can I get permissions for running Kokoro CI tests for this specific folder?

**Comment by kosteev** [comment]
New DataflowTemplatedJobStartOperator requires `apache.beam` extra, I have added it and tests are passing.
PTAL.

File: composer/workflows/airflow_db_cleanup.py [hunk 1]
```diff
@@ -130,7 +130,7 @@
     "keep_last_group_by": None
 }, {
     "airflow_db_model": DagModel,
-    "age_check_column": DagModel.last_scheduler_run,
+    "age_check_column": DagModel.last_parsed_time,
```
Comment (leahecole) [inline composer/workflows/airflow_db_cleanup.py:133]:
```suggestion
    "age_check_column": DagModel.last_parsed_time,  # prior to Airflow 2.0.2 this column was named last_scheduler_run
```

**Review by leahecole — APPROVED** [review]

**Comment by leahecole** [comment]
ready for review #2 when you are @dandhlee 😄

File: composer/workflows/airflow_db_cleanup.py [hunk 2]
```diff
@@ -130,7 +130,7 @@
     "keep_last_group_by": None
 }, {
     "airflow_db_model": DagModel,
-    "age_check_column": DagModel.last_scheduler_run,
+    "age_check_column": DagModel.last_parsed_time,  # prior to Airflow 2.0.2 this column was named last_scheduler_run
```
Comment (dandhlee) [inline composer/workflows/airflow_db_cleanup.py:133]: +1 for mentioning this!

**Review by dandhlee — APPROVED** [review]
LGTM!

**Pull request merged**